docs/runjobs/webui/julia-jupyter.md

```diff
@@ -1,35 +1,27 @@
 # Julia Jupyter
-## Selecting the Julia-Jupyter application
 
-We can use Julia on Jupyter through the [LUMI web interface](https://www.lumi.csc.fi) by selecting the **Julia-Jupyter** application from the menu:
-
-![ood application menu](../../assets/images/julia-jupyter/ood-application-menu.png)
+The Julia-Jupyter app launches Jupyter Lab with support for Julia kernels on a
+LUMI compute node, which you can access from the web interface.
 
 ## Launching Julia-Jupyter
 
-Now, we need to select the resources for running Julia-Jupyter.
-First, we must select a project for billing and partition for computing resources.
-
-![julia jupyter options](../../assets/images/julia-jupyter/options-1.png)
-
-Next, we must set the desired computing resources: CPU cores, memory, local disk, and time.
-
-![julia jupyter options](../../assets/images/julia-jupyter/options-2.png)
+In the form, start by selecting the project for billing, partition for computing resources,
+and the desired computing resources: CPU cores, memory, local disk, and time.
```

Comment:
```suggestion
In the form, start by selecting the project for billing, partition,
and the desired computing resources: CPU cores and time.
```

Reply:
It doesn't seem to be possible to select "memory" and "local disk" anymore

Reply:
The memory dropdown appears on partitions where the memory does not depend on the number of CPU cores, e.g. on the `small` partition. Local disk would appear if the `lumid` partition is selected, but is currently disabled in the web interface.